trunk/SUAVE/Attributes/Solids/Aluminum.py

```diff
@@ -0,0 +1,29 @@
+# Aluminum.py
```

Comment:
File needs doxygen tags and documentation.

Reply:
Added.